Modelica/Clocked/UsersGuide/Clocks.mo

```diff
@@ -49,44 +74,61 @@ is generated:
 </div>
 
 <p>
-As usual in synchronous languages, a clock is represented by a <strong>true</strong> value
-when the clock is active. The relationship between such derived
+The relationship between such derived
 clocks is <strong>exact</strong>, so it is guaranteed that at every 3rd tick of clock
 \"periodicRealClock.y\", the clock \"subSample.y\" is active.
 </p>
 
+<h4>The two types of clocked partitions</h4>
+<p>
+A clocked partition is a set of equations that depend
+on each other and where the boundary variables are marked
+with sample and hold operators (boundary with other clocked partitions
+or with the global continuous-time partition).
+</p>
+<p>
+There are two types of clocked partitions:
+</p>
+<ul>
+    <li>A <strong>clocked discrete-time</strong> partition shall contain
+    <em>no operator relating to continuous time</em>:
+    <strong>der</strong>, <strong>delay</strong>, <strong>spatialDistribution</strong>,
+    no event-related operators (with exception of <strong>noEvent</strong>(&hellip;)),
+    and no <strong>when</strong>-clause with a Boolean condition.
+    A clocked discrete-time partition is thus a standard sampled data system described by difference equations
+    (it can typically contain the <strong>previous</strong> operator).</li>
```

Comment:
```suggestion
    (it can contain calls of the <strong>previous</strong> operator).</li>
```
One could either say that "it typically contain the previous operator" (as it is normal, but not necessary) or "it can contain the previous operator" (as it is allowed); both mixing them doesn't make sense to me and I think "calls of" clarifies it a bit.